readme/product_variant.md

```diff
@@ -0,0 +1,10 @@
+### Model `product_variant`
+
+The `product_variant` subfolder allows to import products provided as CSV files. See the example below:
+
+| id | name | type | categ_id/id 
```

Comment:
Could you add an example with the `lst_price` and `standard_price`, if that's relevant.

Reply:
☝🏻